poh/src/poh_recorder.rs

```diff
@@ -242,6 +242,54 @@ pub enum PohLeaderStatus {
     Reached { poh_slot: Slot, parent_slot: Slot },
 }
 
+struct PohRecorderMetrics {
+    flush_cache_tick_us: u64,
+    flush_cache_no_tick_us: u64,
+    record_us: u64,
+    record_lock_contention_us: u64,
+    report_metrics_us: u64,
+    send_entry_us: u64,
+    tick_lock_contention_us: u64,
+    ticks_from_record: u64,
+    total_sleep_us: u64,
+    last_metric: Instant,
+}
+
+impl PohRecorderMetrics {
+    fn report(&mut self, bank_slot: Slot) {
+        if self.last_metric.elapsed().as_millis() > 1000 {
+            datapoint_info!(
+                "poh_recorder",
+                ("slot", bank_slot, i64),
+                ("flush_cache_tick_us", self.flush_cache_tick_us, i64),
+                ("flush_cache_no_tick_us", self.flush_cache_no_tick_us, i64),
+                ("record_us", self.record_us, i64),
+                (
+                    "record_lock_contention_us",
+                    self.record_lock_contention_us,
+                    i64
+                ),
+                ("report_metrics_us", self.report_metrics_us, i64),
+                ("send_entry_us", self.send_entry_us, i64),
+                ("tick_lock_contention", self.tick_lock_contention_us, i64),
+                ("ticks_from_record", self.ticks_from_record, i64),
+                ("total_sleep_us", self.total_sleep_us, i64),
+            );
+
+            self.flush_cache_tick_us = 0;
```

Comment:
I'm thinking we could add `impl Default for PohRecorderMetrics` and call that here as well as `new_with_clear_signal`

Reply:
done!